### PR TITLE
Support multi-button presses on Philips remotes

### LINKED ISSUE #3591: [BUG] Philips Hue RDM001 simultaneous press not processed
### Bug description

After updating to HA 2024.11 the associated ZHA quirk for Philips Hue Wall Switch RDM001 set in mode 3 (dual rocker) no longer processes simultaneous switching of both switches. When switching both only one of the switches is processed and the second never appears in HA logbook. 

After rolling back init.py and rdm001.py as a custom quirk from commit [c9c9e37](https://github.com/zigpy/zha-device-handlers/commit/c9c9e37395a9c8c1a84bc8e992a413916892e1c9) the behavior now works as before again: switching both switches at the same time will trigger events for left (unnamed) and right switch.

### Steps to reproduce

Switch both switches in mode 3. 
Only one events goes through.

### Expected behavior

Both events go through.

### Screenshots/Video

<details><summary>Screenshots/Video</summary>

[Paste/upload your media here]

</details>

### Device signature

<details><summary>Device signature</summary>

```json
[Paste the device signature here]
```

</details>

### Diagnostic information

<details><summary>Diagnostic information</summary>

```json
[Paste the diagnostic information here]
```

</details>

### Logs

<details><summary>Logs</summary>

```python
[Paste the logs here]
```

</details>

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by fholzer**
## Proposed change
ButtonPressQueue currently resets when receiving a button press from a different button than the previously received one. This prevents proper handling when pressing multiple buttons at the same time.

## Additional information
Fixes #3591 

This is still a draft. Would like to add another test to make sure multiple button presses are handled correctly. So far tests only check for multi-presses of the same button. Also want to wait for feedback from ericu65 in #3591

## Checklist

- [x] The changes are tested and work correctly
- [x] `pre-commit` checks pass / the code has been formatted using Black
- [x] Tests have been added to verify that the new code works

### CODE REVIEW

**Comment by ericu65** [comment]
Manual tested fix from PR and on RDM001 simultaneous presses of both switches while in mode 2 (dual rocker) both events are received correctly now.

File: zhaquirks/philips/__init__.py [hunk 1]
```diff
@@ -188,7 +179,12 @@ class PhilipsRemoteCluster(CustomCluster):
         PressType(SHORT_RELEASE, COMMAND_M_SHORT_RELEASE),
     ]
 
-    button_press_queue = ButtonPressQueue()
+    def __init__(self, endpoint, is_server=True):
+        """Initialize button press queue for each button."""
+
+        super().__init__(endpoint, is_server)
+
+        self.button_press_queue = {k: ButtonPressQueue() for k in self.BUTTONS}
```
Comment (TheJulianJES) [inline zhaquirks/philips/__init__.py]: Let's just remove the two empty lines here.
```suggestion
    def __init__(self, endpoint, is_server=True):
        """Initialize button press queue for each button."""
        super().__init__(endpoint, is_server)
        self.button_press_queue = {k: ButtonPressQueue() for k in self.BUTTONS}
```

File: zhaquirks/philips/__init__.py [hunk 2]
```diff
@@ -288,15 +294,21 @@ def send_press_event(click_count):
                 sim_event_args[ARGS][2] = 2
                 action = f"{button.action}_{press_type.action}"
                 _LOGGER.debug(
-                    "%s - send_press_event emitting simulated action: [%s]",
+                    "%s - send_press_event emitting simulated action: [%s] event_args: %s",
```
Comment (TheJulianJES) [inline zhaquirks/philips/__init__.py]: Adding a comma to the log message here to be similar to others:
```suggestion
                    "%s - send_press_event emitting simulated action: [%s], event_args: %s",
```

**Review by TheJulianJES — APPROVED** [review]
Awesome, thanks!

**Pull request merged**